docs/extensions/logging-and-monitoring.md

```diff
@@ -1,32 +1,39 @@
-# Logging and Monitoring configuration
+# Logging and Monitoring for Extensions
 
-By default, Gardener deploys a central Prometheus, AlertManager, and Grafana instance into the `garden` namespace of all seed clusters.
-Additionally, as part of the shoot reconciliation flow, it deploys a shoot-specific Prometheus, Grafana (and, if configured, an AlertManager) into the shoot namespace next to the other control plane components.
+Gardener provides an integrated logging and monitoring stack for alerting, monitoring and troubleshooting of its managed components by operators or end users. For further information how to make use of it in these roles, refer to the corresponding guides for [exploring logs](https://github.com/gardener/logging/tree/master/docs/usage/logging.md) and for [monitoring with Grafana](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#all-users).
```

Comment:
https://github.com/gardener/logging/tree/master/docs/usage/logging.md does not exist and it is also not part of https://github.com/gardener/logging/pull/58. 

Probably you want to use https://github.com/gardener/gardener/tree/master/docs/usage/logging.md ?

Reply:
It's meant to link to that resource once https://github.com/gardener/logging/pull/58 is merged.
Or if it gets delayed and that's the only left over I will remove this paragraph and add it later, once it's available. Let's see next week.

The https://github.com/gardener/gardener/tree/master/docs/usage/logging.md guide was my first choice too, but its focus happens to be not so much on the end-user/operator "usage" side that I referred here. I think that the guide in the PR above serves better that purpose.

Reply:
yes, i kind of expected logging.md and stuck to it. will fix the link. thx